dbg.py

```diff
@@ -116,6 +129,38 @@ def main():
 	xbox.mem_write(addr, bytes([val]))
 	assert(xbox.mem_read(addr, 1)[0] == val)
 	xbox.free(addr)
+
+	# Inject a function which does `rdtsc; ret`.
+	# RDTSC means "Read Time Stamp Counter". The Time Stamp Counter is a value,
+	# which is incremented every CPU clock cycle.
+	code = bytes([0x0F, 0x31, 0xC3])
+	addr = xbox.malloc(len(code))
+	xbox.mem_write(addr, code)
+
+	# Repeatedly call the injected function until we have a stable timer
+	last_time = None
+	print("Testing call using RDTSC (please wait)")
+	while True:
+
+		# Ask the Xbox for the RDTSC value
+		eax = xbox.call(addr)
+
+		# The timer runs at 733MHz (Xbox CPU Clock speed); We convert it to seconds
```

Comment:
Please wrap this code to < 80 cols

Reply:
Oh, will look into this. It is below 80 columns on my machine as I use 2 spaces per tab.

Reply:
Using tab-size of 4 over here. We should probably just convert this file to use spaces anyway. (Blames self)